### PR TITLE
Adds asciidoctor-mathematical to render math in pdfs

### PR DESCRIPTION
**Pull request by tarikgraba**
Hi,
I have added [asciidoctor-mathematical](https://github.com/asciidoctor/asciidoctor-mathematical) to the gem packages that are installed in the container (full version only).
It is an alternative stem processor that can be used to render math in PDF.

Hopping the patch is and ok and is added soon to the docker file.

T.

### CODE REVIEW

**Comment by tarikgraba** [comment]
@rochdev any chance the PR is reviewed?

**Comment by rochdev** [comment]
I would rather keep the default image at parity with the official asciidoctor image since its purpose is to offer an alpine version of it.

It would be possible however to make a new tag `alpine-asciidoctor:plus` which would contain additional librairies like `asciidoctor-mathematical`.

If you are willing to make it I will gladly accept the contribution :)

Maybe it would even make sense eventually to have some sort of `plus-mini` and `plus-micro` tags, but I would like to at least keep existing tags at parity.

**Comment by tarikgraba** [comment]
Ok, I can try.
What do I have to do exactly to add a tag to the docker hub?
If I understand, I have to add separate dockerfile (in a separate directory) and then?

**Comment by tarikgraba** [comment]
maybe check the last version

File: plus/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+         -LO http://mirrors.ctan.org/fonts/cm/ps-type1/bakoma/ttf/esint10.ttf       \
+         -LO http://mirrors.ctan.org/fonts/cm/ps-type1/bakoma/ttf/eufm10.ttf        \
+         -LO http://mirrors.ctan.org/fonts/cm/ps-type1/bakoma/ttf/msam10.ttf        \
+         -LO http://mirrors.ctan.org/fonts/cm/ps-type1/bakoma/ttf/msbm10.ttf) && \
+    /bin/bash -c "echo sdkman_auto_answer=true > ~/.sdkman/etc/config" && \
+    /bin/bash -c -l "source ~/.sdkman/bin/sdkman-init.sh && sdk install lazybones" && \
+    /bin/bash -c -l "source ~/.sdkman/bin/sdkman-init.sh && sdk flush archives" && \
+    /bin/bash -c -l "source ~/.sdkman/bin/sdkman-init.sh && sdk flush temp" && \
+    apk del bash curl unzip tar gcc g++ make ruby-dev python-dev py-pip libjpeg-turbo-dev zlib-dev zip && \
+    apk del cmake bison flex libffi-dev glib-dev pango-dev gdk-pixbuf-dev libxml2-dev && \
```
Comment (rochdev) [inline plus/Dockerfile]: Can `cairo-dev` be deleted?

Comment (tarikgraba) [reply plus/Dockerfile]: no, I have removed in my last commit

**Comment by tarikgraba** [comment]
:+1:

**Pull request merged**